README.md

```diff
@@ -41,6 +41,31 @@ Follow the instructions for your computer in the wiki:
 - [Linux](https://github.com/massalabs/thyra/blob/main/INSTALLATION.md#linux)
 - [Windows](https://github.com/massalabs/thyra/blob/main/INSTALLATION.md#windows)
 
+### ... manage my wallet?
+1. Create / delete your wallet 
+
+You can access to Thyra wallet interface at URL : <http://my.massa/thyra/wallet/index.html>
+By inputing the 'Nickname' & 'Password', you'll be able to create an encrypted wallet locally on your machine.
+To delete your wallet, simply use the interface. 
+⚠️ If you delete your wallet, you won't be able to edit the website linked to it anymore.
+
+2. Get coins on your wallet
+
+To get coins on your wallet, you have to send your address on [Massa faucet channel](https://discord.com/channels/828270821042159636/866190913030193172)
+Make sure that you use the latest version of Thyra (and defacto Testnet) `here <https://discord.com/channels/828270821042159636/866190913030193172>`_ , otherwise the faucet won't work.
```

Comment:
do you want to put the link of Thyra release or discord faucet?

Reply:
```suggestion
Make sure that you use the latest version of Thyra (and defacto Testnet) [here](https://github.com/massalabs/thyra/releases/latest/), otherwise the faucet won't work.
```

Reply:
Sounds good